exonum/src/runtime/types.rs

```diff
@@ -19,9 +19,9 @@ use std::{borrow::Cow, fmt::Display, str::FromStr};
 
 use crate::{helpers::ValidateInput, proto::schema};
 
-/// Service id type.
+/// Unique service instance identifier.
```

Comment:
If this is the primary doc for this, I'd clarify that:
- This is the secondary identifier, mainly used in transaction messages. The primary is the service instance name.
- The core assigns this identifier when the service is started.